pkg/crc/preflight/preflight_checks_windows.go

```diff
@@ -58,9 +58,28 @@ func checkHyperVInstalled() (bool, error) {
 		return false, errors.New("Hyper-V not installed")
 	}
```

Comment:
I managed to get my system in a state where `checkHyperVInstalled` from this PR succeeded, but `Get-Service vmms` was failing, and `fixHyperVInstalled()` was required in order to fix it, `fixHyperVServiceRunning()` was not enough and was just failing. So I believe `checkHyperVInstalled` needs to keep checking  `Get-Service vmms` is functional, and install hyperv if not. Then `fixHyperVServiceRunning()` can check whether the service is running or not, and start it if it's not running.

Reply:
Haven't tested it yet, but the changes in https://github.com/cfergeau/crc/tree/hyperv-svc should avoid this regression